lib/phoenix_test/playwright.ex

```diff
@@ -38,7 +38,7 @@ defmodule PhoenixTest.Playwright do
   to run tests concurrently in different browsers.
 
   ## Configuration
-  In `config/test.exs`:
+  In `config/runtime.test.exs`:
```

Comment:
It's important that the config that follows occurs at runtime, or else changing `PLAYWRIGHT_HEADLESS`, `PLAYWRIGHT_TRACE`, etc. will have no effect _except_ when the `test.exs` file itself is touched.

Reply:
Are you sure?

`test.exs` is a script file evaluated on every run of `mix test`.
In my experience you only need `runtime.exs` if you want to read environment variables at runtime when running e.g. as a docker image - not via mix.

Are you running your integration tests differently than via `mix test`?

Reply:
Huh! Right you are. I was under the impression it was only reevaluated when the file was changed, but a test locally shows I'm wrong. TIL!

Comment fixed in e1f308b32f0ecf701cfb3fcb9553875fbb01557f